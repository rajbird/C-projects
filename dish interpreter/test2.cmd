echo Salve hoc test II
cd files/moreFiles
sleep 10 &
ls -l [A-Z]*
cd ..
echo another one &
more +19 cap.hero | cut -b 1-10
kjslkdf fjsdklj this will fail
echo language is latin
