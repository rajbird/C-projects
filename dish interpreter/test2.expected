SHELL PID 10352
Salve hoc test II
Child (10356) exitted -- success (0)
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 A
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 Abracadabra
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 C
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 E
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 G
 Even thou

As another
 “If you
 If you ge
 Understan
 However, 

Captain Am
 While som
 In The Fi
 Despite t
” In Cap
” This p
[2]  Done           10362
SHELL PID 10352
Salve hoc test II
Child (10356) exitted -- success (0)
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 A
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 Abracadabra
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 C
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 E
-rw-r--r-- 1 rajbird undergrad 0 Sep 27 14:00 G
 Even thou

As another
 “If you
 If you ge
 Understan
 However, 

Captain Am
 While som
 In The Fi
 Despite t
” In Cap
” This p
[2]  Done           10362
111Child (10367) exitted -- failure (65280)
language is latin
Child (10369) exitted -- success (0)
language is latin
Child (10370) exitted -- success (0)
